src/figuresClasses.test.ts

```diff
@@ -118,7 +116,7 @@ describe('Method "getArea" of class "Circle"', () => {
     const circle = new Circle('red', 6);
 
     expect(circle.getArea())
-      .toBe(113.04);
+      .toBe(113.09);
```

Comment:
The expected area of a circle with radius 6 should be approximately 113.097, not 113.09. Consider using `toBeCloseTo` for floating-point comparisons to avoid precision issues.